ape_ganache/provider.py

```diff
@@ -321,6 +322,12 @@ def get_virtual_machine_error(self, exception: Exception) -> VirtualMachineError
 
         return ContractLogicError(revert_message=message)
 
+    def unlock_account(self, address: AddressType) -> bool:
+        self._make_request("evm_addAccount", [address, ""])
+        self._make_request("personal_unlockAccount", [address, "", 9999999999])
+        self._make_request("evm_setAccountBalance", [address, to_wei(1, "ether")])
+        return True
```

Comment:
This does several things, it does not just unlock the account. Why are we coupling these things here?